modules/setting/actions.go

```diff
@@ -25,5 +26,9 @@ func loadActionsFrom(rootCfg ConfigProvider) {
 		log.Fatal("Failed to map Actions settings: %v", err)
 	}
 
+	actionsSec := rootCfg.Section("actions.artifacts")
+	storageType := actionsSec.Key("STORAGE_TYPE").MustString("")
+
 	Actions.Storage = getStorage(rootCfg, "actions_log", "", nil)
+	Actions.Artifacts = getStorage(rootCfg, "actions_artifacts", storageType, actionsSec)
```

Comment:
Do we need to wait for #23911 ? That's a refactoring PR for `getStorage` , maybe some behavior/design is changed?